lib/clickmeetings/models/account.rb

```diff
@@ -1,5 +1,36 @@
 module Clickmeetings
   class Account < ::Clickmeetings::Model
-    resource_name :accounts
+    set_resource_name "accounts"
+
+    attr_accessor :account_expiration_date,
+                  :meetings_allowed,
+                  :package,
+                  :trainings_allowed,
+                  :account_status,
+                  :api_key,
+                  :city,
+                  :country,
+                  :email,
+                  :firstname,
+                  :lastname,
+                  :state,
+                  :street,
+                  :zip_code,
+                  :username,
+                  :webinars_allowed
+
+    %w(enable disable).each do |m|
+      define_method m do
+        response = client.put(remote_url(__method__, id: id))
+        @account_status = m == "enable" ? "active" : "disabled"
+        self
+      end
+    end
+
+    def conferences
+      response = client.get remote_url(__method__, id: id)
```

Comment:
Can we use something like this:
`Clickmeetings::Conference.find_by(account_id: id)` ?